server/pbench/bin/pbench-verify-s3-backup-tarballs

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import glob
+import hashlib
+import boto3
+import shutil
+import configtools
+
+options = []
+opts, args = configtools.parse_args(options)
+conf, files = configtools.init(opts)
+
+# configuration file
+endpoint_url = conf.get('pbench-server-backup', 'endpoint_url')
```

Comment:
Can you use a dictionary or a simple object to store all these configuration parameters so that we can remove these as global variables and pass them around as one argument?